swig/python/gdal-utils/osgeo_utils/gdal2tiles.py

```diff
@@ -1838,6 +1859,17 @@ def optparse_init() -> optparse.OptionParser:
     )
     p.add_option_group(g)
 
+    # Jpeg options
+    g = optparse.OptionGroup(p, "JPEG options", "Options for JPEG tiledriver")
+    g.add_option(
+        "--jpeg-quality",
+        dest="jpeg_quality",
+        type=int,
+        default=80,
+        help="quality of jpeg image, integer between 1 and 100, default is 75",
```

Comment:
Default is 75 or 80?

Reply:
@jjimenezshaw @zdila is there any progress with this issue?

Reply:
By the way is there an option for creating "mixed" tiles (png in the borders and jpeg in the middle) as is done when using GDAL to create GPKG file?

Reply:
> Default is 75 or 80?

My comment was just about a discrepancy between the default value and the documentation